Assets/SteamVR_Unity_Toolkit/Scripts/Controls/2D/RadialMenu/RadialMenu.cs

```diff
@@ -48,7 +48,7 @@ private void Update()
         //Keep track of pressed button and constantly invoke Hold event
         if (currentPress != -1)
         {
-            buttons[currentPress].Press();
+            Buttons[currentPress].OnHold.Invoke();
```

Comment:
Looks like you're merge is overriding an old change.

`Buttons` was renamed to `buttons` 

Just change all instances of the variable locally in your commit and it and it should be fine and not show as a change.